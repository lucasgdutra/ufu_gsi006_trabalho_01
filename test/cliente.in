joao
1   Arroz   35.99
2   Feijao  12.50