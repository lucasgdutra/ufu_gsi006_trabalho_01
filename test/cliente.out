nome cliente:joao
Id: 1, nome:Arroz, preco:35.99
Id: 2, nome:Feijao, preco:12.50
